twitter/Decorators/DetailsDecorator.h

```diff
@@ -12,13 +12,23 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface DetailsModel : NSObject
+@protocol DetailsDecoratorDelegate
+
+- (void)postReply:(NSString *)tweetId
```

Comment:
Do any other delegates implement a similar method? Main point is that the naming could be more tied to the behavior than to the class it's attached to. ie PostReplying or something like that.